### ISSUE TITLE
Opt-out from data collection

### ISSUE THREAD

**Issue by xuhcc** [nomiclabs/buidler#761]
There should be a way to disable analytics, but I can't find it in the documentation. I guess users have to add `analytics: {enabled: false}}` to their buidler config?

---

Though I'd recommend DNS blocking as a more robust solution. The server to which data is being sent is `o385026.ingest.sentry.io` (source: https://github.com/nomiclabs/buidler/blob/buidler-core-v1.4.4/packages/buidler-core/src/internal/sentry/reporter.ts#L13)

**Comment by alcuadrado** [comment]
That's right @xuhcc, `{analytics: {enabled:false}}` disables it. We'll make it clear in the documentation.

**Comment by xuhcc** [comment]
>A google analytics hit that includes your os, node version, builder version, and the task you are running (custom task names are all replaced with something like "custom-task"). 

Oh, so there's another tracker? 

Worth noting that IP address of a user also becomes known to data collecting services.

---

For others who are reading this thread: data is being sent to `www.google-analytics.com` (source: https://github.com/nomiclabs/buidler/blob/buidler-core-v1.4.4/packages/buidler-core/src/internal/cli/analytics.ts#L40)

**Comment by alcuadrado** [comment]
> Oh, so there's another tracker?

Sentry for errors, google analytics for users. Both can be disabled with the same setting.

> Worth noting that IP address of a user also becomes known to data collecting services.

That's a good point. We'll also document that!

**Comment by xuhcc** [comment]
> {analytics: {enabled:false}} disables it. We'll make it clear in the documentation.
>We'll also make clear what we are sending to the servers. That is:

Hardhat now asks user for consent, that's a good move. But it seems that documentation has not been updated.

**Comment by alcuadrado** [comment]
That setting doesn't exist anymore. It's now a global per-user setting that defaults to false.

**Comment by xuhcc** [comment]
I know, but you also expressed intent to document what data is being collected. It is not particularly important though because the analytics are opt-in now, so feel free to close this issue.

**Comment by alcuadrado** [comment]
Oh, sorry. I thought you were referring to the first part of the comment. We'll make that clear in the docs.

**Comment by CodeSandwich** [comment]
> That setting doesn't exist anymore in the project config. It's now a global per-user setting that defaults to false.

What's the global per-user settings and how can I configure it?

**Comment by nevrending** [comment]
> It's the `telemetry-consent.json` file hidden deep in local configuration.

In case anyone is looking for it, on macOS it's located in `~/Library/Preferences/hardhat-nodejs/telemetry-consent.json`

**Comment by OhadBaehr** [comment]
How do you disable global telemetry?

**Comment by alcuadrado** [comment]
Run `npx hardhat --verbose`, and you are gonna see a log like this

```
  hardhat:core:global-dir Looking up Client Id at /Users/pato/Library/Application Support/hardhat-nodejs/analytics.json +0ms
```

Delete that folder.

**Comment by tallero** [comment]
Hello, I'd like to know what to put in `telemetry-consent.json` to disable telemetry.
